src/plugins/workspace/public/components/data_source_association/association_data_source_modal.scss

```
@@ -0,0 +1,3 @@
+.direct-query-connection-icon {
```

```suggestion
.dsm-associate-modal-direct-query-connection-icon {
```
Can we add some prefix to avoid collision?

---

sure